### DISCUSSION TITLE
CHANGE THE HEIGHT OF TASKBAR

### DISCUSSION THREAD

**Discussion by huuson889** [valinet/ExplorerPatcher#3350] [Ideas]
Infor: Windows 11 
Version 23H2 (OS Build 22631.3593)

I love your app so much, you guys have done a great job.

But it would be more great if you could add:

1. Change the Height of taskbar function like Windhawk App 
2. Use Ctrl + click to open new tab in Explorer windows 11 like Qttabbar App or like Browser

I really like the double click to toggle auto-hide taskbar function and disable Recommendation section in Windows 11 of your App.
Thank you so much and hoping for new functions! @valinet 

https://github.com/valinet/ExplorerPatcher/assets/91197035/d1559d7b-3f14-43c8-842f-bfc8e582d596

**Comment by huuson889** [comment]
Also, add Blur or transparent taskbar function is needed!

**Comment by pyrates999** [comment]
If windhawk app allows you to change the height of the taskbar, use that.

To add transparency, install NiceTaskBar: https://apps.microsoft.com/detail/9pkl2s93xwb5
